Userland/Libraries/LibWeb/Layout/InlineFormattingContext.cpp

```diff
@@ -170,8 +170,8 @@ void InlineFormattingContext::dimension_box_on_line(Box const& box, LayoutMode l
         // FIXME: (10.6.6) If 'height' is 'auto', the height depends on the element's descendants per 10.6.7.
         parent().compute_height(box, AvailableSpace(AvailableSize::make_indefinite(), AvailableSize::make_indefinite()));
     } else {
-        auto container_height = CSS::Length::make_px(m_containing_block_state.content_height());
-        box_state.set_content_height(height_value.resolved(box, container_height).to_px(box));
+        auto inner_height = calculate_inner_height(box, AvailableSize::make_definite(m_containing_block_state.content_height()), height_value);
```

Comment:
i'm not so sure about this. maybe we could just use `m_available_space->height` instead? it seems to work, but i don't know how to test it properly.

Reply:
I think this is okay for now. There might be some scenario where we don't have a height here yet, but I can't think of a way to trigger it right now.